test/powershell/Host/HostUtilities.Tests.ps1

```diff
@@ -46,24 +46,48 @@ Describe "InvokeOnRunspace method as nested command" -tags "Feature" {
     }
 }
 
-Describe "InvokeOnRunspace method on remote runspace" -tags "Feature" {
+Describe "InvokeOnRunspace method on remote runspace" -tags "CI" {
     
     BeforeAll {
-        $wc = [System.Management.Automation.Runspaces.WSManConnectionInfo]::new()
-        $remoteRunspace = [runspacefactory]::CreateRunspace($host, $wc)
-        $remoteRunspace.Open()
+
+        $script:skipTest = $true
+        It "Set up remoting session for test" {
+            $wc = [System.Management.Automation.Runspaces.WSManConnectionInfo]::new()
```

Comment:
I'll skip for travisCI (Linux/OSX) since this is a Windows only remoting test.  Thanks.